media/media.go

```diff
@@ -143,21 +159,34 @@ func (sm svgManager) getFullPath(mediaPath string) string {
 	return filepath.Join(sm.baseDir, mediaPath)
 }
 
-func (sm svgManager) ensureDownloaded(mediaPath string) error {
+func (sm svgManager) ensureDownloaded(mediaPath string, w, h int, c color.Color) (string, error) {
+	// use the colour as a directory to separate different colours of SVG's
+	customizedMediaPath := path.Join(fmt.Sprintf("c%sw%dh%d", colorToHex6(c), w, h), mediaPath)
```

Comment:
In my head it'd be nice if this matched the cache key generation? Seems l like they're doing similar things.

Reply:
Yeah I could re-use that logic for the path

Reply:
The path and cache key are now pretty different because the path no longer needs to include height/width